File: ITSC 3155/itsc-3155/m1/syntax/arrays/arrays.py
# a basic review of arrays (lists) in Python

def main():
    # arrays have no initial size in python
    # we can declare values immediately
    myArray = [1, 2, 3, 4, 5]
    for i in range(10):
        print(f"myArray at index {i} = {myArray[i]}")
        myArray.append((i+1)*2)


if __name__ == "__main__":
    main()

File: ITSC 3155/itsc-3155/m1/syntax/arrays/arrays.java
package m1.syntax.arrays;

public class arrays {
    public static void main(String[] args) {
        int[] myArray = new int[10];
        for (int i = 0; i < 5; i++) {
            myArray[i] = i + 1;
        }
        for (int i = 0; i < 10; i++) {
            // add a value to the array if i > 4
            if (i > 4) {
                myArray[i] = i + 1;
            }
            System.out.println("myArray at index " + i + " = " + myArray[i]);
        }
    }
}